commit e4f7589a3ec285489618ca04c8c0230cc31f3d99
Author: Jim Fehlig <jfehlig@suse.com>
Date:   Wed Nov 24 11:16:38 2021 -0700

    libxl: Modify name of shutdown thread
    
    The current thread name 'ev-<domid>' is a bit terse. Change the name
    to 'shutdown-event-<domid>', allowing it to be distinguished between
    thread handling other event types.
    
    Signed-off-by: Jim Fehlig <jfehlig@suse.com>
    Reviewed-by: Daniel P. Berrangé <berrange@redhat.com>
    Reviewed-by: Ján Tomko <jtomko@redhat.com>

Index: libvirt-7.10.0/src/libxl/libxl_domain.c
===================================================================
--- libvirt-7.10.0.orig/src/libxl/libxl_domain.c
+++ libvirt-7.10.0/src/libxl/libxl_domain.c
@@ -678,7 +678,7 @@ libxlDomainEventHandler(void *data, libx
         shutdown_info->driver = driver;
         shutdown_info->vm = vm;
         shutdown_info->event = (libxl_event *)event;
-        name = g_strdup_printf("ev-%d", event->domid);
+        name = g_strdup_printf("shutdown-event-%d", event->domid);
         /*
          * Cleanup will be handled by the shutdown thread.
          */
